### PR TITLE
#161342804 Create article functionality

### PR DESCRIPTION
**Pull request by victor-nweze-deel**
#### What does this PR do?
- Enables a user create articles

#### Description of Task to be completed?
- Create component for creating articles
- Incorporate tinymce WYSIWYG editor
- Configure tinymce to automatically upload images to cloudinary

#### How should this be manually tested?
- Git clone this repository
- Cd into the haven-ah-frontend folder
- Run `npm run start:dev`
- Hit the create article button from the landing page

#### What are the relevant pivotal tracker stories?
- [#161342804](https://www.pivotaltracker.com/story/show/161342804)

#### Screenshots (if appropriate)
screen shot 2018-12-11 at 2 19 03 pm

### CODE REVIEW

**Comment by ucheg6** [comment]
Good work bro @ProfJigsaw 👍

File: src/components/containers/createArticle/createArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+              `,
+              height: 400,
+              resize: true,
+              paste_data_images: true,
+              inline: true,
+              images_upload_handler: (blobInfo, success, failure) => {
+                const formData = new FormData();
+                formData.append('upload_preset', 'default_preset');
+                formData.append('file', blobInfo.blob(), blobInfo.filename());
+                axios({
```
Comment (TheoOkafor) [inline src/components/containers/createArticle/createArticle.jsx:166]: Nice

**Review by TheoOkafor — COMMENTED** [review]
Nice work @ProfJigsaw 
👍

File: src/components/views/CreateArticlePage.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Navbar from '../containers/navbar/navbar';
+import CreateAticle from '../containers/createArticle/createArticle';
+
+const CreateArticlePage = () => {
+  return (
+    <div className="layout">
+      <div className="container">
+        <header>
+          <Navbar isLoggedIn />
```
Comment (seyi-adeleke) [inline src/components/views/CreateArticlePage.jsx:10]: why are you hardcoding isLoggedIn ?

Comment (victor-nweze-deel) [reply src/components/views/CreateArticlePage.jsx:10]: Yeah, Theo had that. I was waiting for his branch to be merged in.

**Pull request merged**